### PR TITLE
[validate] - Fixed an issue with empty .pack-ignore with newlines

### PR DESCRIPTION
**Pull request by GuyAfik**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-8355

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Changelog
 ## Unreleased
 * Fixed an issue where **validate** falsely failed with error `DS108` on descriptions ending with brackets that contains a dot at the end of them.
+* Fixed an issue where the **validate -g** failed reading a `.pack-ignore` file that was previously empty with newlines.
```
Comment (tkatzir) [inline CHANGELOG.md:4]: @ShirleyDenkberg , please review.
```suggestion
* Fixed an issue where the **validate -g** failed reading a `.pack-ignore` file that was previously only contained newlines.
```

File: demisto_sdk/commands/validate/tests/validators_test.py [hunk 2]
```diff
@@ -3189,6 +3189,15 @@ def test_validate_no_disallowed_terms_in_customer_facing_docs_end_to_end(repo, m
...
         ),
+        (
+            {"Packs/test/.pack-ignore"},
+            "[file:test.yml]\nignore=BA108\n",
+            b"\n",
+            "",
+            {
+                "Packs/test/Integrations/test/test.yml",
+            },
+        ),
```
Comment (tkatzir) [inline demisto_sdk/commands/validate/tests/validators_test.py:3200]:
```suggestion
        (
            {"Packs/test/.pack-ignore"},
            "[file:test.yml]\nignore=BA108\n",
            b"\n",
            "",
            {
                "Packs/test/Integrations/test/test.yml",
            },
        ),
        (
            {"Packs/test/.pack-ignore"},
            "[file:test.yml]\nignore=BA108\n",
            b"\n\n",
            "",
            {
                "Packs/test/Integrations/test/test.yml",
            },
        ),
```

File: demisto_sdk/commands/validate/tests/validators_test.py [hunk 3]
```diff
@@ -3207,6 +3216,7 @@ def test_get_all_files_edited_in_pack_ignore(
     - Case 3: pack-ignore mocks where each file is pointed to a different integration yml.
     - Case 4: old .pack-ignore that is empty and current .pack-ignore that was updated with ignored validation
     - Case 5: old .pack-ignore which is not in the remote branch repo, but only exist in the local branch
+    - Case 6: old empty .pack-ignore which contains newlines.
```
Comment (tkatzir) [inline demisto_sdk/commands/validate/tests/validators_test.py:3219]:
```suggestion
    - Case 6: old empty .pack-ignore which contains a newline.
    - Case 7: old empty .pack-ignore which contains newlines.
```

File: demisto_sdk/commands/validate/validate_manager.py [hunk 4]
```diff
@@ -1999,7 +1999,10 @@ def get_all_files_edited_in_pack_ignore(self, modified_files: set) -> set:
             # if the repo does not have remotes, get the .pack-ignore content from the master branch in Github api
             # if the repo is not in remote / file cannot be found from Github api, try to take it from the latest commit on the default branch (usually master/main)
             old_pack_ignore_content = get_remote_file(old_file_path, "master")
-            if old_pack_ignore_content == b"":  # found as empty file in remote
+            if (
+                isinstance(old_pack_ignore_content, bytes)
+                and old_pack_ignore_content.strip() == b""
+            ):  # found as empty file in remote
```
Comment (tkatzir) [inline demisto_sdk/commands/validate/validate_manager.py:2005]:
```suggestion
            ):  # found an empty file in remote
```

**Review by tkatzir — APPROVED** [review]
Great!
Approved, with minor comments.

**Comment by ShirleyDenkberg** [comment]
@tkatzir Doc review completed.

**Pull request merged**